homeassistant/components/withings/config_flow.py

```diff
@@ -83,27 +73,9 @@ async def async_oauth_create_entry(self, data: dict[str, Any]) -> FlowResult:
             )
 
         if self.reauth_entry.unique_id == user_id:
-            self.hass.config_entries.async_update_entry(self.reauth_entry, data=data)
+            self.hass.config_entries.async_update_entry(
+                self.reauth_entry, data={**self.reauth_entry.data, **data}
+            )
             return self.async_abort(reason="reauth_successful")
 
         return self.async_abort(reason="wrong_account")
-
-
-class WithingsOptionsFlowHandler(OptionsFlowWithConfigEntry):
-    """Withings Options flow handler."""
-
-    async def async_step_init(
-        self, user_input: dict[str, Any] | None = None
-    ) -> FlowResult:
-        """Initialize form."""
-        if user_input is not None:
-            return self.async_create_entry(
-                data=user_input,
-            )
-        return self.async_show_form(
-            step_id="init",
-            data_schema=self.add_suggested_values_to_schema(
-                vol.Schema({vol.Required(CONF_USE_WEBHOOK): bool}),
```

Comment:
Is there a translation string to clean up?

Reply:
Good point